newdle/client/src/components/answer/answer.module.scss

```diff
@@ -6,15 +6,23 @@
   text-align: center;
 }
 
-.date {
+.date:global(.ui.header) {
   text-align: center;
+  padding-bottom: 5px;
+  padding-top: 5px;
+
+  &.date-selected {
+    background-color: rgba($turquoise, 1);
```

Comment:
isn't that the same as just `background-color: $turquoise`? (opacity 1 is fully visible)

Reply:
uh, yep, too many tests